Tools/Scripts/webkitpy/style/checkers/cpp.py

```diff
@@ -2815,6 +2815,44 @@ def check_ismainthread(filename, clean_lines, line_number, file_state, error):
     error(line_number, 'runtime/ismainthread', 4, "Use 'isMainRunLoop()' instead of 'isMainThread()' in Source/WebKit.")
 
 
+def check_wtf_make_span(clean_lines, line_number, file_state, error):
+    """Looks for use of 'Span { ... }' which should be replaced with 'WTF::makeSpan()'.
```

Comment:
Don’t we also want to check for use of `std::span`? Would be good for portability to warn about that too.

Reply:
Great point.  I was being a bit too forward-looking.